delfin/task_manager/manager.py

```diff
@@ -39,6 +49,49 @@ def __init__(self, service_name=None, *args, **kwargs):
         self.alert_sync = alerts.AlertSyncTask()
         super(TaskManager, self).__init__(*args, **kwargs)
 
+    def periodic_performance_collect(self):
+        LOG.info("Scheduled performance collection starting.")
+
+        # Load the scheduler configuration file
+        try:
+            with open(CONF.scheduler.config_path) as f:
+                data = json.load(f)
+        except FileNotFoundError as e:
+            raise exception.ConfigNotFound(e)
+
+        # create the object of periodic scheduler
+        schedule = BlockingScheduler()
```

Comment:
Trying to understand, this will run inside the delfin process and as it is scheduled periodic collection, making it Blocking Scheduler will be the only thing running inside process right? Why didn't we choose BackgroundScheduler here?

Reply:
Changed to BackgroundScheduler(). Thanks

Reply:
thanks